packages/m/mkl/xmake.lua

```diff
@@ -3,90 +3,96 @@ package("mkl")
     set_homepage("https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html")
     set_description("Intel® oneAPI Math Kernel Library")
 
+    add_configs("runtime",      {description = "Set MKL runtime for gcc/clang like compilers.", default = "default", type = "string", values = {"default", "custom"}})
```

Comment:
runtime is builtin configuration, please rename to `mkl_runtime`

Reply:
Sorry, I had not understood this command in the OpenMP package. At this point, there is no real need for this setup in MKL in my opinion, especially considering how complicated the configuration of the linker is.  This should be set through the other parameters, so I have removed the reference.